garden/src/features/users/components/model-deployments/StatusHeader.tsx

```diff
@@ -0,0 +1,63 @@
+import { useState } from "react";
+import { InfoIcon } from "lucide-react";
+import {
+    Tooltip,
+    TooltipContent,
+    TooltipProvider,
+    TooltipTrigger,
+} from "@/components/shadcn/tooltip";
+
+export interface StatusColors {
```

Comment:
I think the status column might end up being a YAGNI - it's not clear to me that we'll have cause to show anything but deployed apps here.